### PR TITLE
Klohman usability recommendations

### PR DESCRIPTION
**Pull request by klohman**
updated homepage, about, buyers guide, sellers guide, comparison with recommendations from usability testing

### CODE REVIEW

File: src/content/sellers-guide/index.mdx [hunk 1]
```diff
@@ -12,12 +12,16 @@ import ContentListItem from "@components/ContentListItem.astro"
 We are currently in the process of developing the Sellers' guide, which will help you manage your contracts, get on OASIS+ and find resources to get the most out of your experience. 
 Below is a preview of the guide topics. 
 
-<Alert heading="Upcoming dates and events">
-- Current solicitation status: Closed on October, 20, 2023. 
-- Next proposal submission opportunity: No later than FY2025; they will be continuously open from that point forward.
-
-Visit the [OASIS+ Interact Community for updates.](https://buy.gsa.gov/interact/community/196/activity-feed)
-</Alert>
+   <USWSDAlert heading="Updates and announcements">
```
Comment (weiwang-gsa) [inline src/content/sellers-guide/index.mdx:15]: You probably need to change USWDSAlert back to Alert as the import statement give alias name as Alert. Or you can change the top import statement from "import Alert from "@components/USWDSAlert.astro;" to "import USWDSAlert from "@components/USWDSAlert.astro;"

**Review by weiwang-gsa — CHANGES_REQUESTED** [review]
Looks good, just for the sellers-guide/index.mdx, this is markdown page, component already support the markdown syntax including links properly render as internal or external icon, so no need to put html code inside the component. This page is different from pages/index.astro, index.astro is astro page and use the basic html code. I highlighted the before and after, for bullet point and link, you can reference the original code for syntax. I highlight the portion you can reference to. Let me know if you need any help. 
image

**Comment by klohman** [comment]
> Looks good, just for the sellers-guide/index.mdx, this is markdown page, component already support the markdown syntax including links properly render as internal or external icon, so no need to put html code inside the component. This page is different from pages/index.astro, index.astro is astro page and use the basic html code. I highlighted the before and after, for bullet point and link, you can reference the original code for syntax. I highlight the portion you can reference to. Let me know if you need any help. image

@weiwang-gsa I think I've fixed this but would appreciate a look!

**Review by weiwang-gsa — APPROVED** [review]
LGTM

**Pull request merged**